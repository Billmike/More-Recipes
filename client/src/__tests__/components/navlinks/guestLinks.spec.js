import React from 'react';
import { shallow } from 'enzyme';
import GuestLinks from '../../../components/navlinks/guestLinks';

describe('<guestLinks />', () => {
  it('Should mount the guestlink component properly', () => {
    const wrapper = shallow(<GuestLinks />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('ul').length).toBe(1);
    expect(wrapper.find('li').length).toBe(3);
  });
});
