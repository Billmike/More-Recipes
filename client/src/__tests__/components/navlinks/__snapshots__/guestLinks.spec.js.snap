// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<guestLinks /> Should mount the guestlink component properly 1`] = `
<ul
  className="nav navbar-nav ml-auto"
>
  <li
    className="nav-item active"
  >
    <Link
      className="nav-link custom-link"
      replace={false}
      to="/"
    >
      Home
      <span
        className="sr-only"
      >
        (current)
      </span>
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link custom-link"
      id="signupID"
      replace={false}
      to="/register"
    >
      Register
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link custom-link logout"
      id="signinID"
      replace={false}
      to="/login"
    >
      Login
    </Link>
  </li>
</ul>
`;
